M6-Backend/FacilityInterfaces/interfaceRate.js

```diff
@@ -0,0 +1,14 @@
+
+function interfaceRate(rateScore){
+   if(rateScore.length == 0){
+    return 0;
+   }
+    const testQuery = []
```

Comment:
Codacy has a fix for the issue: [Replace `····const·testQuery·=·[]` with `··const·testQuery·=·[];`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
  const testQuery = [];
```